src/controllers/post.js

```diff
@@ -1,15 +1,26 @@
-import { sendDataResponse } from '../utils/responses.js'
+import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 import { PrismaClient } from '@prisma/client'
 const prisma = new PrismaClient()
 
 export const create = async (req, res) => {
   const { content } = req.body
+  const { id } = req.user
 
   if (!content) {
-    return sendDataResponse(res, 400, { content: 'Must provide content' })
+    return sendMessageResponse(res, 400, 'Must provide content')
   }
 
-  return sendDataResponse(res, 201, { post: { id: 1, content } })
+  try {
+    const createdPost = await prisma.post.create({
+      data: {
+        content,
+        userId: id
+      }
+    })
+    return sendDataResponse(res, 201, { post: createdPost })
+  } catch (err) {
+    return sendMessageResponse(res, 500, err.message)
+  }
```

Comment:
Best not to send raw errors back to the client, it may contain details that should be kept private (like the structure of your database tables, queries that prisma runs). Take a look at the `create` function in `src/controllers/user.js` to see how that try / catch is implemented